File: palindrome.js
function palindromeChecker(str){
    //splits string between letters so then it can be reversed and rejoined
    var splitString = str.split("");
    //reversed and rejoined
    var reversedString = splitString.reverse().join("");
    //compare the reverse string to original parameter
    if(reversedString == str){
        console.log(str, "is a palindrome")
    }
    else{
        console.log(str, "is not a palindrome")
    }
}
//variables created
let var1 = "OHO"
let var2 = "NOT A PALINDROME"
let var3 = "12345654321"
console.log("Reversing 2 strings")
//function called with variables
palindromeChecker(var1);
palindromeChecker(var2);
palindromeChecker(var3);

File: palindrome.cpp
#include <iostream>
#include <algorithm>
#include <string>
using std::string;
//function to reverse
void palindromeChecker(string str){
  // new variable is equal to input
  string reversedString = str;
  // new variable is reversed using a built in function
  std::reverse(reversedString.begin(), reversedString.end());
  //comparing reversed to input parameter
  if(reversedString == str){
    std::cout << str << "is a palindrome" << std::endl;
  }
  else
    std::cout << str << "is not a palindrome" << std::endl;

}

int main(){
  //variables to use declared
  string var1 = "OHO";
  string var2 = "Not A PALINDROME";
  string var3 = "12345654321";
  std::cout << "Reversing 2 strings"<< std::endl;
  //calling the function
  palindromeChecker(var1);
  palindromeChecker(var2);
  palindromeChecker(var3);

}
